### PR TITLE
Use event.key over event.keyCode/which

### PR DESCRIPTION
**Pull request by mc10**
There are a few `keyCode` instances that I did not change, as they were trickier to modify correctly.

### CODE REVIEW

File: lib/modules/commentTools.js [hunk 1]
```diff
@@ -1078,24 +1078,6 @@ function lod() {
 	});
 } */
 
-export const KEYS = {
```
Comment (andytuba) [inline lib/modules/commentTools.js]: Can we keep this as `NAMED_KEYS = { SPACE: ' ', ENTER: 'Enter', ESCAPE: 'Escape', ... }` for stronger validation?

Comment (mc10) [reply lib/modules/commentTools.js]: Sure, sounds good to me.

**Comment by andytuba** [comment]
Thanks! Neat little refactor.

**Comment by erikdesjardins** [comment]
This needs testing in Edge (which I can do at some point), since apparently it has "non-standard key identifiers"... https://caniuse.com/#feat=keyboardevent-key

**Comment by mc10** [comment]
Good catch; seems like the issue is [fixed but not yet released](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8860571/). We'll need to add a [shim similar to this](https://github.com/shvaikalesh/shim-keyboard-event-key/blob/master/index.js) for Edge to work.

**Comment by BenMcGarry** [comment]
Depending when we expect this 5.12 to land we may be able to just avoid the shim. The fix should be released April timeish.

**Comment by mc10** [comment]
I added a polyfill for Edge, so this should be mergeable. I don't have Edge to test with though.

**Comment by erikdesjardins** [comment]
Tested in Edge 16
